include/vrv/att.h

```diff
@@ -266,6 +266,25 @@ class AttComparison {
     ClassId m_classId;
 };
 
+//----------------------------------------------------------------------------
+// InterfaceComparison
+//----------------------------------------------------------------------------
+
+class InterfaceComparison {
```

Comment:
Make the class inherit from AttComparison (we should actually rename AttComparison just Comparison because it can be use for any type of matching. I will do that once the PR is merged.)

Reply:
ClassId and InterfaceId are separate enums with separate checking methods, though. As it is having InterfaceComparison inherit from AttComparison would just add functionality for searching by class to InterfaceComparison. Is this desirable?

Reply:
You are right, I missed the ClassId. We whould have the following hierarchy:
```
class Comparison
class AttComparison : public Comparison
class InterfactComparison : public Comparison
```
And the the method / functor / functorparams acting on Comparison. Then we need only one set of method / functor / functorparams. OK?

Reply:
Ok. I'll make a commit implementing this.